contracts/HybridCustody.cdc

```diff
@@ -271,6 +271,18 @@ pub contract HybridCustody {
         // capabilities.
         pub var filter: Capability<&{CapabilityFilter.Filter}>?
 
+        // display is its own field on the ChildAccount resource because only the parent should be able to set this
+        // field.
+        pub let childAccountsDisplay: {Address: MetadataViews.Display}
+
+        pub fun setDisplay(address: Address, _ d: MetadataViews.Display) {
```

Comment:
I'm inclined to name this `setChildAccountDisplay()` for consistency with corresponding getter, `getChildAccountDisplay()` - alternatively `set/getChildDisplay()`. Either way, consistency between the two is helpful.